api/src/app/modules/project/public-notice.service.ts

```diff
@@ -177,8 +187,27 @@ export class PublicNoticeService extends DataService<PublicNotice, Repository<Pu
     response.revisionCount = entity.revisionCount;
     response.operationStartYear = entity.operationStartYear;
     response.operationEndYear = entity.operationEndYear;
+		response.postDate = entity.postDate;
     return response;
   }
 
-}
-
+	// Override: for some dto property business values transformation.
```

Comment:
Why is this needed?

Reply:
Sorry, the comment line is confusing. I should remove that.
But now I know why I override convertDto() method from dataService. Not sure if you have time for discussion (for design decision).